package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@bugsnag/react-native": "^7.4.0",
     "@react-native-community/async-storage": "^1.12.0",
+    "@react-native-community/cli": "^4.14.0",
```

Comment:
Was necessary for me to run `npx react-native run-android`, maybe you have this  installed globally?

Not sure why it caused such a huge `package-lock` diff!